packages/trader/src/Modules/Contract/Containers/contract.jsx

```diff
@@ -19,6 +19,10 @@ class Contract extends React.Component {
     }
 
     render() {
+        if (!/\d+/.test(this.props.match.params.contract_id)) {
```

Comment:
```suggestion
        if (!/^\d+$/.test(this.props.match.params.contract_id)) {
```

Reply:
what about isNaN()?

Reply:
Thanks for the suggestion. Yes, I think isNaN would be a more readable choice.